swap_meet/clothing.py

```diff
@@ -1,2 +1,7 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition=0, age=0):
+        super().__init__(category, condition, age)
```

Comment:
Great! Taking advantage of inheritance by using the parent class's constructor makes this concise!